src/main/java/core/basesyntax/IsoscelesTrapezoidSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoidSupplier {
+    private static final RandomGeometricValue randomSize = new RandomGeometricValue();

```

Comment:
The 'randomSize' variable might be better named 'randomGeometricValueSupplier' for clarity and to follow naming conventions.

Reply:
I guess not, for example:
Figure: isosceles trapezoid, area: 4.282561245E8 sq. units, first side: 33065.0 units, second side: 274.0 units, height: 25691.0 units, color: RED